test/e2e/multi-node-3.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $BASEDIR
+
+user=admin
+password=pass
+
+couch1dir=$(mktemp -d -t couchdb-2x-XXXXXXXXXX)
+couch2dir=$(mktemp -d -t couchdb-2x-XXXXXXXXXX)
+couch3dir=$(mktemp -d -t couchdb-2x-XXXXXXXXXX)
+mkdir -p $couch2dir/shards $couch2dir/.shards $couch3dir/shards $couch3dir/.shards
+jsondataddir=$(mktemp -d -t json-XXXXXXXXXX)
+
+export DB1_DATA=$couch1dir
+export DB2_DATA=$couch2dir
+export DB3_DATA=$couch3dir
+export CHT_NETWORK=couch-migration-network-cluster
+export COUCHDB_USER=$user
+export COUCHDB_PASSWORD=$password
+export COUCH_PORT=25984
+export COUCH_CLUSTER_PORT=25986
+export HOST_COUCH_URL=http://$user:$password@127.0.0.1:$COUCH_PORT
+export COUCH_URL=http://$user:$password@couchdb-1.local:$COUCH_PORT
+
+# cleanup from last test, in case of interruptions
+docker rm -f -v scripts-couchdb-1.local-1 scripts-couchdb-2.local-1 scripts-couchdb-3.local-1
```

Comment:
Minor: consider using `trap` for cleanup of containers and temporary directories

Reply:
I think we should add this in a separate PR if you don't mind. Please feel free to create an issue.

Reply:
Sure, any minor comments I consider optional, if it bothers me too much I can create a PR at some later point :)